### PR TITLE
Update README.md with project members

### PR DESCRIPTION
**Pull request by JawadHossain**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,42 @@
...
+# Contributing
 
 Send a pull request and be sure to update this file with your name.
 
-Contributors / Licensing
-========================
+# Contributors / Licensing
+
 
 Generally everything is LICENSE'D under the Apache 2 license by Abram Hindle.
```
Comment (tylerelston) [inline README.md:26]: Should our names be included here since it will be covering our code too?

Comment (JawadHossain) [reply README.md:26]: I think it would make sense to add it there too.

**Review by olivier-vadiaval — APPROVED** [review]
Looks good to me. We can add more details later on.

**Review by olivier-vadiaval — APPROVED** [review]
This works too.

**Review by tylerelston — APPROVED** [review]

**Review by debecerra — APPROVED** [review]

**Review by mxiuwang — APPROVED** [review]

**Pull request merged**